src/manifest.json

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
I kind of feel like this belongs at the root of the repo as well.

Reply:
Yes, it would make it consistent with other Docker repos.